layouts/partials/fragments/section-header.html

```diff
@@ -0,0 +1,41 @@
+{{- $bg := .Params.background | default "light" }}
+
+{{ "<!-- Section Header -->" | safeHTML }}
+<section id="{{ .Name }}">
+  <div class="overlay container-fluid
+    {{- printf " bg-%s" $bg -}}
+  ">
+    <div class="container pt-5 pb-0">
```

Comment:
pt-5 might be too much. It feels like a section header doesn't need so much spacing. What do you think?